chapters/en/slides/module5/module5_06.Rmd

```diff
@@ -0,0 +1,288 @@
+
+We can see that most contributions seem to be donated in the middle of the week
+and the Wed stands out as the day
+where the highest amount of money is donated.
+
+To draw additional attention to this bar,
+we could use color to highlight it.
+
+---
+
+## Color can be used to highlight specific features of a visualization
+
+```{python}
+alt.Chart(donations).mark_bar().encode(
+    alt.Y('week_day', sort=list(day_abbr), title=None),
+    alt.X('sum(sum)', axis=alt.Axis(format='$s'), title='Total donated amount'),
+    color = alt.condition(alt.datum.week_day == 'Wed',
+                          alt.value('coral'),
+                          alt.value('steelblue')))
+```
+
+Notes:
+To add a highlight color,
+we could create a new column in our dataframe
+that has a unique value for the bars we want to highlight
+and the same value for everything else.
+
+Then we could use this
+as a categorical column
+to color by via the `color` encoding.
+
+The downside of this is that we need to modify the dataframe,
+which might not always be desired.
+It is also clunky when we are computing an aggregation in Altair,
+such as using this `sum` here,
+and we would need to recompute this in pandas.
+
+Because of these inconveniences,
+it is often preferred to use the built-in Altair approach for highlighting
+by passing a conditional encoding to the `color` parameter of the chart
+as we do in this slide.
+
+Conceptually,
+you can think of `alt.condition` like this:
+`alt.condition(IF this is true, DO this, ELSE DO this)`.
+
+So here we are checking if the `week_day` column has the value `'Wed'`,
+and if it does we use the `'coral'` color,
+otherwise we use the default `'steelblue'` color.
+
+We need to `alt.value` to tell Altair that we want to use this specific color name,
+rather than looking for a column in the dataframe with this name.
+
+`alt.datum` is a way to unambiguously access a column name in conditional statements
+and other Altair computations,
+so that it is not confused for just that string of letters.
+
+---
+
+## Text labels can be added with custom annotations
+
+```{python}
+chart = alt.Chart(donations).mark_bar().encode(
+    alt.Y('week_day', sort=list(day_abbr), title=None),
+    alt.X('sum(sum)', axis=alt.Axis(format='$s'), title='Total donated amount'),
+    color = alt.condition(alt.datum.week_day == 'Wed',
+                          alt.value('coral'),
+                          alt.value('steelblue')))
+(chart 
+ + chart
+ .mark_text(align='left', dx=-125, dy=-15)
+ .encode(text=alt.condition(alt.datum.week_day == 'Wed', alt.value('Salaries are paid on Wed'), alt.value(''))))
+```
+
+Notes:
+We can add a custom text annotation to the plot
+in the same way that we added a custom color.
+This could be an explanatory label,
+for example when an event happened on a time axis,
+or as the label that we made up for this chart.
+
+We use `dx` and `dy` to position the label
+relative the top of the bar.
```

Comment:
Can we narrate the code a bit more here? What is the new mark? How do we add it? etc.